{{cookiecutter.github_repository_name}}/docs/api/users.md

```diff
@@ -19,44 +25,16 @@ Content-Type application/json
 201 Created
 
 {
-  "id": 1,
-  "first_name": "Richard",
-  "last_name": "Hendriks",
-  "auth_token": "fFBGRNJru1FQd44AzqT3Zg",
-  "email": "hendriks@piepiper.com"
+  "id": "6d5f9bae-a31b-4b7b-82c4-3853eda2b011",
+  "username": "richard",
+  "password": "pbkdf2_sha256$24000$aGozcCr6QXhv$WCgPt2voqVO+Nno2flVnNnLcfks6Yq8XJyxoadB/r50=",
+  "auth_token": "132cf952e0165a274bf99e115ab483671b3d9ff6"
 }
 ```
-## Get a user's profile information
-
-**Request**:
-
-`GET` `users/:id`
-
-Parameters:
-
-Name | Type | Description
----|---|---
-id | integer | The id associated with the user object.
```

Comment:
Yeah so `id` is part of the path, not a parameter so removed the parameters section.

It should be clear from the context that `id` is a uuid, right?